packages/rulesets/src/spectral/test/xms-pageable-for-list-calls.test.ts

```diff
@@ -56,14 +56,35 @@ test("XmsPageableForListCalls should find errors if x-ms-pagebale property is no
           },
         },
       },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Music/Configurations/config1": {
```

Comment:
This is a point get since it ends in a resource name (`config1`). So, it should not have `value`, `nextLink`, or `x-ms-pageable`